Pipfile

```diff
@@ -10,6 +10,9 @@ prio = {path = ".", editable = true}
 pytest = "*"
 coverage = "*"
 pytest-cov = "*"
+setuptools = "*"
+wheel = "*"
```

Comment:
This should be removed

Reply:
Can you expand on this comment? Why is it included here now and when should it be removed?

Reply:
I originally planned to build the package as a wheel, but pypi doesn't allow uploading packages with the linux tag that `bdist_wheel` produces in the docker image. I'll remove it in a second pass of this PR where I'll upload the package to the real pypi at the end.